src/fft.rg

```diff
@@ -144,133 +152,435 @@ function fft.generate_fft_interface(itype, dtype)
     return iface.get_tunable(DEFAULT_TUNABLE_NODE_COUNT)
   end
 
+  -- Get the number of GPUs
```

Comment:
Remove. This is entirely specified by the name of the task, and in fact the task name here is actually *more* informative than the comment because it specifies that these are *local* GPUs.

Reply:
Removed